include/seastar/util/sampler.hh

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef SRC_PROFILING_MEMORY_SAMPLER_H_
+#define SRC_PROFILING_MEMORY_SAMPLER_H_
+
+#include <stdint.h>
+
+#include <atomic>
+#include <random>
+
+#include "perfetto/ext/base/utils.h"
```

Comment:
This won't work

Reply:
You have probably realized at this point but I just want to clarify the approach we took here. 

We got advised that when importing only a part of a library it's best to make it clear what the original imported work is and what changes were made on top. 

Hence, we split the import & change into two. The first commit (what you commented on and what will definitely not compile) just imports the original file as is. The second commit then does the modifications.

If you prefer any other way of doing this we are of course very happy to change.

Reply:
No, it's fine and a good approach.

Reply:
A line in the patch changelog would help (for reviewers that read the changelog).

Reply:
Have amended the commit text